src/components/Profile/EditProfile/EditProfile.js

```diff
@@ -0,0 +1,274 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Container,
+  Grid,
+  Paper,
+  TextField,
+  Fab,
+  Button,
+  IconButton,
+  Typography,
+  Modal,
+  Backdrop,
+  Avatar,
+  Box,
+} from "@material-ui/core";
+import { CameraAlt, Close } from "@material-ui/icons";
+import "roboto-fontface";
```

Comment:
Can we move this to the App.js or any root file rather than importing it everywhere?

Reply:
Alright sir. I will move it